### ISSUE TITLE
multilinguality

### ISSUE THREAD

**Issue by gregorycrane** [OpenGreekAndLatin/cts_leipzig_ui#1]
How do we add a mechanism for localizing to different languages. This should initially be (1) English, (2) German, (3) French, (4) Italian, (5) Croatian, (6) Persian, (7) Russian, (8) Arabic. We will need to think about what happens as the UI evolves and we integrate things like xlations and dictionaries, which will be available at varying levels for different languages - this is not just website localization.

But let's get started  with the problem.

**Comment by PonteIneptique** [comment]
Totally agree. This should go with the new design

**Comment by PonteIneptique** [comment]
To explain the reasoning : If we spend time doing i18n right now, we will need to do it all over again after. It would be much more cost efficient to do these as one package.

**Comment by gregorycrane** [comment]
I agree -- I just want to make it clear that we always keep this in mind and look for the right moment to get started. If we start too soon, we waste effort. Same if we start too late.
